.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.8
+  python: python3
```

Comment:
I'm not sure it's a good idea (I think it's just the same as removing it) but let's try

Reply:
ah sorry, I can put it back, I now installed python3.8 on my mac

Reply:
```suggestion
  python: python3.8
```